packages/terra-table/src/Table.module.scss

```diff
@@ -52,11 +68,13 @@
     }
 
     tbody {
+      background-color: var(--terra-table-tbody-background-color);
+
       tr {
-        border-top: var(--terra-table-row-border-horizontal, 1px solid #c8cacb);
+        border-top: var(--terra-table-row-border-top, 1px solid #c8cacb);
```

Comment:
hmmmmm. If we roll with refactoring `cell` to `td` and `th` above, it would make sense to be consistent and use `tr` instead of `row`.  We would have to refactor the rest of the row theme variables. Thoughts?

```scss
border-top: var(--terra-table-tbody-tr-border-top, 1px solid #c8cacb);
```

Reply:
I'm okay with sticking with the `cell` / `row` naming convention. We can re-evaluate if we need more granular theming for tables if that comes up, but for now, I think this is good.